acceptance/tests/ticket_2455_on_solaris_init_provider_should_start_service_in_own_smf_contract.rb

```diff
@@ -112,15 +114,15 @@
       if agent.is_pe?
 
         step "Stop puppet on #{agent}"
-          on(agent, "svcadm disable #{agent_service};sleep 70;svcadm disable #{agent_service}")
+          on(agent, "svcadm disable pe-puppet;sleep 70;svcadm disable pe-puppet")
 
         step "Stop fixture service on #{agent}"
           on(agent, "#{fixture_service_stop}")
 
         step "Enable puppet service on #{agent}"
-          on(agent, "svcadm enable #{agent_service};sleep 10") do
-            puppet_ctid = on(agent, "svcs -Ho CTID #{agent_service} | awk '{print $1}'").stdout.chomp.to_i
-            service_ctid = on(agent, "ps -eo ctid,args | grep #{fixture_service} | grep -v grep | awk '{print $1}'").stdout.chomp.to_i
+          on(agent, "svcadm enable pe-puppet;sleep 10") do
+            puppet_ctid = on(agent, "svcs -Ho CTID pe-puppet | awk '{print $1}'").stdout.chomp.to_i
+            service_ctid = on(agent, "ps -eo ctid,args | grep pe-puppet | grep -v grep | awk '{print $1}'").stdout.chomp.to_i
 
```

Comment:
This line, to extract `service_ctid`, changed from grepping for `#{fixture_service}` to grepping for `pe-puppet` but it seems like the step below is still assuming `#{fixture_service}` for the comparison. Was this correct?

I don't fully get the context for this test, so apologies if I'm just missing the boat :)